addons/nuxeo-core-binarymanager-cloud/nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/blob/s3/S3BlobStoreConfiguration.java

```diff
@@ -225,11 +226,11 @@
     public final boolean metadataAddUsername;
 
     /**
-     * The retention mode with which the bucket is configured.
+     * Is Object Lock feature enabled at s3 level.
      *
-     * @since 11.4
+     * @since 2021.13
      */
-    public final ObjectLockRetentionMode bucketRetentionMode;
```

Comment:
Deprecate just in case, as we're in 10.10?

Reply:
We agreed with @efge to remove these public constants as they were unlikely used by anyone